loader/icd_dispatch_generated.c

```diff
@@ -17,8 +17,11 @@
  */
 
 #include "icd_dispatch.h"
+#include "icd_envvars.h"
 #include "icd.h"
 
+#include <string.h>
+
 #ifdef __cplusplus
```

Comment:
This file should be reverted.
```suggestion
#include "icd_dispatch.h"
#include "icd.h"

#ifdef __cplusplus
```

Reply:
fixed